api_tests/e2e/rfc003/btc_usdt/test.js

```diff
@@ -0,0 +1,58 @@
+const chai = require("chai");
+chai.use(require("chai-http"));
+const omnilayer = require("../../../lib/omnilayer.js");
+const actor = require("../../../lib/actor.js");
+const should = chai.should();
+
+const alice = actor.create("alice", {});
+const bob = actor.create("bob", {});
+
+const bob_final_address = "mzNFGtxdTSTJ1Lh6fq5N5oUgbhwA7Nm7cA";
+
+const alpha_asset = 3;
+const beta_asset = 4200;
+
+describe("RFC003: Bitcoin for Reg Test Omni Token (USD Tether style)", () => {
+    before(async function() {
+        this.timeout(50000);
+        await omnilayer.activateSegwit();
+        await alice.wallet.omni().btcFund(1);
+        await omnilayer.generate();
+        await bob.wallet.omni().btcFund(1);
+        await omnilayer.generate();
+    });
+
+    let tokenId;
+    it("Create Regtest Omni Token", async function() {
+        const res = await alice.wallet.omni().createOmniToken();
+        res.propertyid.should.be.a("number");
+        tokenId = res.propertyid;
+    });
+
+    let aliceOmniUTXO;
+    it("Grant Regtest Omni Token", async function() {
+        const grantAmount = beta_asset * 3;
+        aliceOmniUTXO = await alice.wallet.omni().grantOmniToken(tokenId, alice.wallet.omni().identity().output, grantAmount);
+        const balance = await omnilayer.getBalance(tokenId, alice.wallet.omni().identity().address);
+        balance.should.equal(grantAmount.toString());
```

Comment:
Let's add this to ensure Bob has no tokens
```suggestion
        balance.should.equal(grantAmount.toString());
        
        const bob_token_balance = await omnilayer.getBalance(tokenId, bob.wallet.omni().identity().address);
        bob_token_balance.should.equal("0");
```